satellite/metainfo/validation.go

```diff
@@ -316,6 +316,13 @@ func (endpoint *Endpoint) validatePointer(ctx context.Context, pointer *pb.Point
 				return err
 			}
 
+			// expect that too much time has not passed between order limit creation and now
+			latestCommitTime := limit.OrderCreation.Add(endpoint.maxCommitInterval)
+			if latestCommitTime.Before(time.Now()) {
```

Comment:
If I am not wrong, the above two lines can be replaced with a simpler check:
```go
time.Since(limit.OrderCreation) <= endpoint.maxCommitInterval
```

Reply:
Done